airbyte-integrations/connectors/destination-databend/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "destination_databend"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-databend-sqlalchemy = "==0.1.6"
+databend-sqlalchemy = "==0.4.7"
```

Comment:
Because of this change, it is breaking existing connector. Probably a change in how client is built.
You can check running:
```shell
poetry install --with dev  
poetry run destination-databend spec                                                                                                                                                                                                                                                                     
```

Reply:
Thanks! Have done.